Sources/Shared/UI/Documents/DocumentsViewController.m

```diff
@@ -227,6 +229,7 @@ - (void)dealloc
     _dicContentOfFolder = nil;
     
```

Comment:
Some variables need to be released or set to `nil`
- `fileToApplyAction`
- `popoverPhotoLibraryController`
- `_stringForUploadPhoto`
- `popoverProperties`